LVST/Program.cs

```diff
@@ -113,6 +113,8 @@ private static async Task<bool> FindAndUseChromecast()
         private static async Task<Stream> StartTorrenting(Options cliOptions)
         {
             var engine = new ClientEngine();
+            await engine.RegisterDhtAsync (new MonoTorrent.Dht.DhtEngine (MonoTorrent.Dht.Listeners.DhtListenerFactory.CreateUdp (5555)));
```

Comment:
This is a bit crappy... but... :)

Reply:
Don't know much about DHT but I trust you :-)
Can make the port configurable later. Been thinking of a `LibVLCSharp.MonoTorrent` nuget.

Reply:
That works for me! The port isn't super important as long as it doesn't clash with an already in use port. 

You need to forward it using upnp or nat-pmp (or manually) for some aspects of the DHT capabilities, but lookups for peers should work just fine without an open port so this is a net win